src/Components/Cart/CartTotalPriceTable.js

```diff
@@ -1,23 +1,36 @@
+import { useCart } from "../../Context/DataContext/CartContext"
 export const CartTotalPriceTable = () =>{
+    const {cart} = useCart()
+    
+    const totalPrice = cart.item.reduce((acc, cv) => {
+        return acc + Number(cv.price)*cv.qty}
+    ,0)
+
+    const totalDiscount = cart.item.reduce((acc, cv) =>{
+        return acc +(cv.price*(cv.discount/100))*cv.qty
```

Comment:
If cv.price doesn't need numeric conversion at line10, would it be needed at line6? Please check.